greenheart/simulation/technologies/steel.py

```diff
@@ -0,0 +1,970 @@
+        ].tolist()[0]
+    else:
+        price_breakdown_O2sales = 0
+
+    if gen_inflation > 0:
+        price_breakdown_taxes = (
+            price_breakdown_taxes
+            + price_breakdown.loc[
+                price_breakdown["Name"] == "Capital gains taxes payable", "NPV"
+            ].tolist()[0]
+        )
+
+    # Calculate financial expense associated with equipment
+    price_breakdown_financial_equipment = (
+        price_breakdown.loc[
+            price_breakdown["Name"] == "Repayment of debt", "NPV"
+        ].tolist()[0]
+        + price_breakdown.loc[
+            price_breakdown["Name"] == "Interest expense", "NPV"
+        ].tolist()[0]
+        + price_breakdown.loc[
+            price_breakdown["Name"] == "Dividends paid", "NPV"
+        ].tolist()[0]
+        - price_breakdown.loc[
+            price_breakdown["Name"] == "Inflow of debt", "NPV"
+        ].tolist()[0]
+        - price_breakdown.loc[
+            price_breakdown["Name"] == "Inflow of equity", "NPV"
+        ].tolist()[0]
+    )
+
+    # Calculate remaining financial expenses
+    price_breakdown_financial_remaining = (
+        price_breakdown.loc[
+            price_breakdown["Name"] == "Non-depreciable assets", "NPV"
+        ].tolist()[0]
+        + price_breakdown.loc[
+            price_breakdown["Name"] == "Cash on hand reserve", "NPV"
+        ].tolist()[0]
+        + price_breakdown.loc[
+            price_breakdown["Name"] == "Property tax and insurance", "NPV"
+        ].tolist()[0]
+        - price_breakdown.loc[
+            price_breakdown["Name"] == "Sale of non-depreciable assets", "NPV"
+        ].tolist()[0]
+        - price_breakdown.loc[
+            price_breakdown["Name"] == "Cash on hand recovery", "NPV"
+        ].tolist()[0]
+    )
+
+    price_breakdown_check = (
+        price_breakdown_eaf_casting
+        + price_breakdown_shaft_furnace
+        + price_breakdown_oxygen_supply
+        + price_breakdown_h2_preheating
+        + price_breakdown_cooling_tower
+        + price_breakdown_piping
+        + price_breakdown_elec_instr
+        + price_breakdown_buildings_storage_water
+        + price_breakdown_misc
+        + price_breakdown_installation
+        + price_breakdown_labor_cost_annual
+        + price_breakdown_labor_cost_maintenance
+        + price_breakdown_labor_cost_admin_support
+        + price_breakdown_maintenance_materials
+        + price_breakdown_water_withdrawal
+        + price_breakdown_lime
+        + price_breakdown_carbon
+        + price_breakdown_iron_ore
+        + price_breakdown_hydrogen
+        + price_breakdown_natural_gas
+        + price_breakdown_electricity
+        + price_breakdown_slag
+        + price_breakdown_taxes
+        + price_breakdown_financial_equipment
+        + price_breakdown_financial_remaining
+        + price_breakdown_O2sales
+    )  # a neater way to implement is add to price_breakdowns but I am not sure if ProFAST can handle negative costs
```

Comment:
Again, profast provides a price breakdown, so I'm not sure why all this code pulling the individual parts.

Reply:
I'm not really sure about this either, should we remove it?